elasticsearch-log/config.ini

```diff
@@ -1,6 +1,25 @@
-#example for config
-[elasticsearch-log]
-insightFinder_license_key =
-insightFinder_project_name =
-insightFinder_user_name =
-sampling_interval =
+[insightfinder]
+license_key =
+project_name =
+user_name =
+sampling_interval = 1
+if_http_proxy =
+if_https_proxy =
+
+[reporting_parameters]
+serverurl =
+reporting_interval = 1
+
+[elastic_search]
+elasticsearchhost = localhost
+elasticsearchport = 9200
+elasticsearchindex = syslog
+timefieldname = @timestamp
+istimestamp = true
+hostnamefield = host
+dateformatinjodatime = yyyy-MM-dd'T'HH:mm:ss
+dateformatinstrptime = %Y-%m-%dT%H:%M:%S
```

Comment:
make sure to pass raw=True when grabbing both date field format options to avoid parameter expansion with "%"

Reply:
Added raw = True for dateformatinjodatime and dateformatinstrptime.